package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "opentelemetry-contrib",
-  "version": "0.13.0",
+  "version": "0.13.1",
   "description": "This is a repository for OpenTelemetry JavaScript contributions.",
-  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "repository": "https://github.com/open-telemetry/opentelemetry-js-contrib",
```

Comment:
I think this is not correct. if you want to use the full url you have to add also type and append .git
see https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository

for github repos using just user/repo is fine.

Reply:
user/repo format broke the tool I use to generate the changelog